server/util/util.go

```diff
@@ -0,0 +1,99 @@
+package util
```

Comment:
:+1: for having this in a separate package. Maybe `testutils` is a better package name? Then this file could be `http.go`.